lib/staggered_oprod.cu

```diff
@@ -182,15 +182,14 @@ namespace quda {
   template<typename real, typename Output, typename InputA, typename InputB>
   __global__ void interiorOprodKernel(StaggeredOprodArg<real, Output, InputA, InputB> arg)
     {
+      using complex = complex<real>;
+      using matrix = Matrix<complex,3>;
```

Comment:
`nColor` comment as before. iirc this is the first time I've seen a `using matrix = Matrix<complex,3>;` in the `*_oprod.cu` files, let's propagate that elsewhere while we're making this global change (unless there's a reason to not)